clang/include/clang/Basic/Builtins.td

```diff
@@ -4554,6 +4554,13 @@ def HLSLWaveActiveCountBits : LangBuiltin<"HLSL_LANG"> {
   let Prototype = "unsigned int(bool)";
 }
 
+// HLSL
+def HLSLWaveGetLaneIndex : LangBuiltin<"HLSL_LANG"> {
+  let Spellings = ["__builtin_hlsl_wave_get_lane_index"];
+  let Attributes = [NoThrow, Const];
```

Comment:
We probably want to add support for adding `Convergent` here.

Reply:
Yes, but this implies touching to the mangling pattern..

Reply:
We should remove all the explicit convergent markers. We should go forward with https://reviews.llvm.org/D69498. Once you have convergence in the language at all, every function must be assumed convergent by default